js/other-discount-code-button.js

```diff
@@ -0,0 +1,65 @@
+jQuedry(document).ready(function($){
```

Comment:
Why is this a separate file?

I was looking at the applydiscountcode JS code earlier and realize that we should be able to abstract this to a function so we don't have so much redundant code.

Reply:
this needs to be deleted. Shouldn't have been here.